test/unit/helpers/ConfigHelper.ts

```diff
@@ -90,7 +85,15 @@ test(`promptOptions 4 - autoRegister = true backwards compatibility for custom i
   };
   (fakeUserConfig as any).promptOptions = {
     slidedown: {
-      enabled: true,
+      prompts: [{
```

Comment:
Should not update this test with the new format. Should keep the old userConfig format